packages/job-dispatch/src/policies/release-sequencing.ts

```diff
@@ -45,6 +47,26 @@ export const isPassingNoActiveJobsPolicy: ReleaseIdPolicyChecker = async (
           `),
         ),
       ),
+    )
+    .orderBy(
+      asc(schema.deployment.id),
+      desc(schema.release.createdAt),
+      desc(schema.release.version),
```

Comment:
does order matter here? Do we want to use schema.release.createdAt

Reply:
probably not I can remove

Reply:
since we do it below